src/many/Cargo.toml

```diff
@@ -35,6 +35,7 @@ pkcs8 = { version = "0.8", features = [ "pem", "std", "encryption" ] }
 many-macros = { path = "../many-macros" }
 openssh-keys = "0.5.0"
 rand = "0.8.4"
+rand-07 = { package = "rand", version = "0.7"}  # Version compatible with ed25519-dalek
```

Comment:
This looks like it should be in `dev-dependencies`?

Reply:
Fixed